test/runner/lib/cli.py

```diff
@@ -614,24 +626,30 @@ def add_environments(parser, tox_version=False, tox_only=False):
                               action='store_true',
                               help='run from the local environment')
 
-    if tox_version:
-        environments.add_argument('--tox',
-                                  metavar='VERSION',
-                                  nargs='?',
-                                  default=None,
-                                  const='.'.join(str(i) for i in sys.version_info[:2]),
-                                  choices=SUPPORTED_PYTHON_VERSIONS,
-                                  help='run from a tox virtualenv: %s' % ', '.join(SUPPORTED_PYTHON_VERSIONS))
+    if data_context().content.is_install:
```

Comment:
In various places in the code there are two conditions like this that are used:

- `if data_context().content.is_install:` - When we need to know if we're testing the Ansible source, without caring what kind of content is being tested when we're not.
- `if data_context().content.collection:` - When we need to know that we're testing an Ansible collection.

Currently these two conditions are the inverse of each other. However, if we ever add support for testing pre-collection roles (something I included in an early prototype but left out of this PR), then the difference matters, as we would then end up adding a third condition:

- `if data_context().content.role:` - When we need to know that we're testing an Ansible role.

As for the name `is_install`, at one point I actually named it `is_ansible`, but thought `is_install` was better since it matched the `INSTALL_ROOT` constant. If you still think it makes sense to rename it, I can think of a few options and I'm open to other suggestions:

- `is_ansible`
- `is_source`
- `is_ansible_install`
- `is_ansible_source`

Reply:
Personally I prefer `is_ansible` or `is_ansible_install` as it makes it more obvious the `is_install` is referring to Ansible's install location and not the collection path. I'm not married to it but I definitely had to look at the code to make sure that is correct.